src/common/element-scroll/test/test.browser.js

```diff
@@ -0,0 +1,34 @@
+const expect = require('chai').expect;
+const elementScroll = require('../');
+
+describe('element-scroll', () => {
+    const contentDiv = document.createElement('div');
+
+    for (let index = 0; index < 10; index += 1) {
+        const innerDiv = `<div class="item${index}" style="height: 50px">content</div>`;
+        contentDiv.innerHTML += innerDiv;
+    }
+
+    const innerEl1 = contentDiv.querySelectorAll('.item5')[0];
+    const innerEl2 = contentDiv.querySelectorAll('.item2')[0];
```

Comment:
I'd suggest renaming these like `secondItemEl fifthItemEl`
Can also use `querySelector()` in place of `querySelectorAll()[0]`